app/api/items/delete_item.feature

```diff
@@ -0,0 +1,144 @@
+    And the database has the following table 'permissions_granted':
+      | group_id | item_id | is_owner | source_group_id |
+      | 10       | 22      | true     | 10              |
+      | 11       | 21      | false    | 10              |
+    And the database has the following table 'groups_groups':
+      | parent_group_id | child_group_id |
+      | 10              | 11             |
+    And the groups ancestors are computed
+    And the database has the following table 'attempts':
+      | id | participant_id | root_item_id |
+      | 0  | 10             | null         |
+      | 1  | 10             | 22           |
+    And the database has the following table 'answers':
+      | participant_id | attempt_id | item_id | author_id | created_at          |
+      | 10             | 0          | 21      | 10        | 2019-05-30 11:00:00 |
+      | 10             | 0          | 22      | 10        | 2019-05-30 11:00:00 |
+      | 10             | 1          | 21      | 10        | 2019-05-30 11:00:00 |
+    And the database has the following table 'filters':
+      | id | user_id | item_id |
+      | 1  | 10      | 21      |
+      | 2  | 10      | 22      |
+      | 3  | 11      | null    |
+    And the database has the following table 'groups_contest_items':
+      | group_id | item_id |
+      | 10       | 21      |
+      | 11       | 22      |
+    And the database has the following table 'item_dependencies':
+      | item_id | dependent_item_id |
+      | 21      | 21                |
+      | 21      | 22                |
+      | 22      | 21                |
+    And the database has the following table 'items_ancestors':
+      | ancestor_item_id | child_item_id |
+      | 20               | 21            |
+      | 20               | 22            |
+      | 21               | 22            |
+    And the database has the following table 'items_items':
+      | parent_item_id | child_item_id | child_order |
+      | 20             | 21            | 1           |
+      | 21             | 22            | 1           |
+    And the database has the following table 'items_strings':
+      | item_id | language_tag |
+      | 20      | fr           |
+      | 21      | fr           |
+      | 22      | en           |
+      | 22      | fr           |
+    And the database has the following table 'results':
+      | attempt_id | participant_id | item_id | result_propagation_state |
+      | 0          | 10             | 21      | done                     |
+      | 0          | 10             | 22      | done                     |
+      | 1          | 10             | 21      | done                     |
+    And the database has the following table 'threads':
+      | id | creator_id | type | item_id |
+      | 1  | 10         | Help | 21      |
+      | 2  | 10         | Help | 22      |
+      | 3  | 11         | Bug  | null    |
+    And the database has the following table 'languages':
+      | tag |
+      | sl  |
```

Comment:
Looks not possible (for FK) to have 'en' and 'fr' in items_strings if it is not in this one.

Reply:
FK checks are disabled for the 'the database has the following table' step. Do you want me to fix this anyway?

Reply:
fixed